docs/resources/Channel.md

```diff
@@ -74,7 +74,8 @@ Represents a guild or DM channel within Discord.
   "nsfw": true,
   "topic": "24/7 chat about how to gank Mike #2",
   "last_message_id": "155117677105512449",
-  "parent_id": "399942396007890945"
+  "parent_id": "399942396007890945",
+  "default_auto_archive_duration": 240
```

Comment:
240 isn't a valid choice, right?
1hr, aka `60` would probably be better

Reply:
I have no idea where I got 240 from, I think I meant 1440 for 1 day since that seems to be the default, although I've noticed if the channel hasn't ever changed it, this doesn't even get sent in the payload. Thus making 60 the better example?

Reply:
If you changed it to 60 and then back to 1440 it'd probably send down 1440 in the payload
But yeah, if it's never been set before it won't get sent down, no need to waste space on it